pytorch_lightning/utilities/deprecation.py

```diff
@@ -38,33 +38,36 @@ def deprecated(target: Callable, ver_deprecate: str = "", ver_remove: str = "")
     """
     Decorate a function or class ``__init__`` with warning message
      and pass all arguments directly to the target class/method.
-     """
+    """
 
-    def inner_function(func):
+    def inner_function(base):
 
-        @wraps(func)
+        @wraps(base)
         def wrapped_fn(*args, **kwargs):
             is_class = inspect.isclass(target)
             target_func = target.__init__ if is_class else target
             # warn user only once in lifetime
-            if not getattr(inner_function, 'warned', False):
+            if not getattr(wrapped_fn, 'warned', False):
                 target_str = f'{target.__module__}.{target.__name__}'
-                func_name = func.__qualname__.split('.')[-2] if is_class else func.__name__
+                base_name = base.__qualname__.split('.')[-2] if is_class else base.__name__
                 rank_zero_warn(
-                    f"The `{func_name}` was deprecated since v{ver_deprecate} in favor of `{target_str}`."
+                    f"The `{base_name}` was deprecated since v{ver_deprecate} in favor of `{target_str}`."
```

Comment:
This `The` sounds weird to me

```suggestion
                    f"`{base_name}` was deprecated since v{ver_deprecate} in favor of `{target_str}`."
```